src/components/admin/app-navbar/AppNavbar.vue

```diff
@@ -6,10 +6,10 @@
     <span slot="center">
       {{$t('navbar.messageUs')}}&nbsp;<a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
     </span>
-    <message-dropdown/>
-    <notification-dropdown/>
-    <language-dropdown/>
-    <profile-dropdown>
+    <message-dropdown @toggle-menu="hideSidebar()"/>
+    <notification-dropdown @toggle-menu="hideSidebar()"/>
+    <language-dropdown @toggle-menu="hideSidebar()"/>
+    <profile-dropdown @toggle-menu="hideSidebar()">
```

Comment:
onResize toggles sidebard different (desktop -> mobile, mobile -> desktop) than in vuestic.epicmax.co